qt_ui/windows/basemenu/QBaseMenu2.py

```diff
@@ -246,10 +247,14 @@ def closeEvent(self, close_event: QCloseEvent):
         GameUpdateSignal.get_instance().updateGame(self.game_model.game)
 
     def get_base_image(self):
-        if self.cp.cptype == ControlPointType.AIRCRAFT_CARRIER_GROUP:
-            return "./resources/ui/carrier.png"
-        elif self.cp.cptype == ControlPointType.LHA_GROUP:
-            return "./resources/ui/lha.png"
+        game = self.game_model.game
```

Comment:
is this used anywhere?

Reply:
Nope, leftovers from the earlier upgrade_to_supercarrier call in QBaseMenu2.py
This has now been removed.